src/browser-realm.ts

```diff
@@ -185,7 +207,7 @@ export default function createSecureEnvironment(distortionMap?: Map<RedProxyTarg
     // In Chrome debugger statements will be ignored when the iframe is removed
```

Comment:
this comment is now duplicated with the one in the removeIframe function.

Reply:
removed